frontend/src/components/pages/CampOverview/CampDetails.tsx

```diff
@@ -88,8 +88,8 @@ const CampDetails = ({ camp, setCamp }: CampDetailsProps): JSX.Element => {
     setCamp({ ...camp, campCounsellors });
   };
 
-  const handleVolunteerChange = (e: any) => {
-    setCamp({ ...camp, volunteers: e.target.value });
+  const handleVolunteerChange = (newVolunteers: string) => {
```

Comment:
Not sure why this change got pulled in- it's not part of my changes. Should I remove this from the PR?

Reply:
Hmm its not in dev, is this one of your own changes from another ticket in progress? If so, I guess it'll be more organized to remove it.

Reply:
reverted it back to dev's file